packages/yew/Cargo.toml

```diff
@@ -14,7 +14,7 @@ keywords = ["web", "webasm", "javascript"]
 categories = ["gui", "wasm", "web-programming"]
 description = "A framework for making client-side single-page apps"
 readme = "../../README.md"
-rust-version = "1.56.0"
+rust-version = "1.60.0"
```

Comment:
The MSRV has been updated to 1.60 due to the requirement of namespaced dependency.